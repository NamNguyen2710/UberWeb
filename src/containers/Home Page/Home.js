import homeImg from '../../images/uber-home-img.jpg'

function Home() {
  return (
    <div className="homepage">
      <div className="white-box">
        <div className="box">
          <img src={homeImg} alt="uber-img"/>
          <div>
            <h2>Your ride, on demand</h2>
            <p>Whether you're headed to work, the airport, or put on the town, Uber connects you with a reliable ride in minutes. One tap and a car come directly to you.</p>
          </div>
          <button className="black-btn">Register with iPhone</button>
        </div>
      </div>
    </div>
  );
}

export default Home;