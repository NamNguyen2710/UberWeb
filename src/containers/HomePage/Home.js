import homeImg from "../../images/uber-home-img.jpg";
import router from "../../router.js";
import { useHistory } from "react-router-dom";

function Home() {
  let history = useHistory();
  function handelClick() {
    history.push(router.SIGNUP);
  }

  return (
    <div className="homepage">
      <div className="white-box">
        <div className="box">
          <img src={homeImg} alt="uber-img" />
          <div>
            <h2>Your ride, on demand</h2>
            <p>
              Whether you're headed to work, the airport, or put on the town,
              Uber connects you with a reliable ride in minutes. One tap and a
              car come directly to you.
            </p>
          </div>
          <button className="black-btn" onClick={handelClick}>
            Register with iPhone
            <span className="inline-arrow">â†’</span>
          </button>
        </div>
      </div>
    </div>
  );
}

export default Home;
