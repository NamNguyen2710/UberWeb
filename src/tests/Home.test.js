import React from "react";
import renderer from "react-test-renderer";
import Home from "../containers/HomePage/Home";

describe("Home", () => {
  it("should render", () => {
    const snapshot = renderer.create(<Home />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
