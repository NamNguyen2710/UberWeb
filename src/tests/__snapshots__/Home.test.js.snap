// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render 1`] = `
<div
  className="homepage"
>
  <div
    className="white-box"
  >
    <div
      className="box"
    >
      <img
        alt="uber-img"
        src="uber-home-img.jpg"
      />
      <div>
        <h2>
          Your ride, on demand
        </h2>
        <p>
          Whether you're headed to work, the airport, or put on the town, Uber connects you with a reliable ride in minutes. One tap and a car come directly to you.
        </p>
      </div>
      <button
        className="black-btn"
        onClick={[Function]}
      >
        Register with iPhone
      </button>
    </div>
  </div>
</div>
`;
